platform/database/database.go

```diff
@@ -30,7 +39,35 @@ var DefaultConfig = Config{
 	Database: "postgres",
 }
 
-func Connect(config Config) (*sqlx.DB, error) {
+var DefaultMetrics = Metrics{
+	QueryLatencyMs:  metrics.Float64("sql_query_latency_ms", "The query latency in milliseconds", metrics.UnitMilliseconds),
+	QueryErrorTotal: metrics.Float64("sql_query_error_total", "The query error total", metrics.UnitDimensionless),
+}
+
+var MetricViews = []*metrics.View{
+	&metrics.View{
+		Name:        "sql/query_latency",
+		Measure:     DefaultMetrics.QueryLatencyMs,
+		Description: "The distribution of the latencies",
+		Aggregation: metrics.ViewDistribution(0, 25, 100, 200, 400, 800, 10000),
+	},
+	&metrics.View{
+		Name:        "sql/query_error",
+		Measure:     DefaultMetrics.QueryErrorTotal,
+		Description: "The number of errors",
+		Aggregation: metrics.ViewCount(),
+	},
+}
+
+type DB struct {
+	db *sqlx.DB
+}
```

Comment:
I'm not a huge fan of `DB.db`. However, the only thing that I have in mind is `QueryDriver.conn` and `TransactionDriver.conn` but I'm also not convinced by that... 😓

So -> 🤷‍♂️

Reply:
> I'm sorry but after writing the comment above, this db.db.DB made me laugh. 😂

Yes that's why I named it `db`.
Basically it wraps a `sqlx.DB` struct that wraps a `sql.DB`  struct. Even if the name is redundant it's only used inside the package so I don't think it's a huge issue.

Moreover name like `conn` is misleading because [`sql.Conn`](https://pkg.go.dev/database/sql?tab=doc#Conn) exists and is different. It's really a wrapper around [`sql.DB`](https://pkg.go.dev/database/sql?tab=doc#DB)